IBPSA/Airflow/Multizone/Examples/ReverseBuoyancy.mo

```diff
@@ -264,6 +263,11 @@ bottom floor directly exchanges air with the outside.
 </html>", revisions="<html>
 <ul>
 <li>
+May 15, 2019, by Jianjun Hu:<br/>
+Replaced fluid source FixedBoundary with Boundary_pT. This is for 
+<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1072\"> #1072</a>.
```

Comment:
Please replace this with
```
Replaced fluid source. This is for 
<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1072\">#1072</a>.
```
or use the full class name for the new model with ```<a href=\"modelica://....```. Otherwise if we rename in the future `Boundary_pT` then this would not be updated, and we have no means to check for invalid class names.